0.17/files/docker-entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -x
```

Comment:
This also left the image in an un-runnable state.

Reply:
Probably need to use ash or I just install bash.